manifests/interface.pp

```diff
@@ -194,9 +199,28 @@
   $bridge          = undef,
   $arpcheck        = undef,
 
-  # Suse specific
+  ## Suse specific
   $startmode       = '',
-  $usercontrol     = 'no'
+  $usercontrol     = 'no',
+  $ethtool_opts    = undef,
+  $firewall        = undef,
+  $aliases         = undef,
+  $remote_ipaddr   = undef,
+
+  # For bonding
+  $bond_master     = undef,
```

Comment:
Uhm, I just realized tha some of these params are already existing in the defining. Would you please verify them and remove the duplicates?